### DISCUSSION TITLE
Is the project dead?

### DISCUSSION THREAD

**Discussion by RAMChYLD** [DistroAV/DistroAV#735] [General]
Starting to get concerned since OBS-NDI is part of my streaming workflow. The NDI5 rewrite appears to have stalled for 5 months (last commit was late last year) while the current branch received it's latest commit last year.

**Comment by AnthonyHunt** [comment]
I REALLY hope it isn't!

About 3 months before COVID-19 lockdown in the UK my city suffered from extraordinary flooding that meant my students couldn't access campus. We made an OBS/NDI workflow to allow them to work and be live contributors broadcasting news remotely. They were sceptical about how we were making it work. Questioning it's place in industry (NDI).

Roll on COVID-19 and literally the next day saw our exact workflow being implemented by major broadcasters. Full credibility restored instantly. Since then my students understanding of NDI, workflow and switchers has gone through the roof.

So, an unadulterated plea to the community; Please don't let this project die! Please support it in any way you can! 

@Palakis - what can we do to help? I truly understand that post pandemic the world has become very busy again - but what can the world do to help you continue this project? 

Thanking you forever!

Anthony.

**Comment by Trouffman** [comment]
short answer : No.
